stdlib/public/TensorFlow/Tensor.swift

```diff
@@ -664,7 +665,7 @@ public extension Tensor {
   @inlinable @inline(__always)
   @differentiable(wrt: self where Scalar : TensorFlowFloatingPoint)
   func reshaped(to newShape: TensorShape) -> Tensor {
-    return reshaped(toShape: Tensor<Int32>(newShape.dimensions))
+    return reshaped(toShape: Tensor<Int32>({newShape.dimensions.map(Int32.init)}()))
```

Comment:
Since this is a workaround for an AD issue, please add a TODO here for removing the workaround.

Reply:
Added TODO with link to [TF-433](https://bugs.swift.org/browse/TF-433).